Source/ablastr/fields/IntegratedGreenFunctionSolver.cpp

```diff
@@ -71,7 +69,9 @@ computePhiIGF ( amrex::MultiFab const & rho,
     amrex::Real const dy = cell_size[1];
     amrex::Real const dz = cell_size[2];
 
-    obc_solver->setGreensFunction(
+    if (!is_igf_2d_slices){
+        // 2D sliced solver
```

Comment:
This comment here and the one in the else part are incorrect.

Reply:
oops! I will fix it